phpunit/functional/TicketTest.php

```diff
@@ -6416,7 +6416,27 @@ protected function timelineItemsProvider(): iterable
 
         $postonly_user_id = getItemByTypeName(\User::class, 'post-only', true);
         $normal_user_id   = getItemByTypeName(\User::class, 'normal', true);
-        $tech_user_id     = getItemByTypeName(\User::class, 'tech', true);
+        $tech_user     = getItemByTypeName(\User::class, 'tech');
+        $tech_user_id = $tech_user->getID();
+
+        $tech_p = getItemByTypeName(\Profile::class, 'Technician');
+        $new_tech = $tech_p->clone();
+
+        $profileright = new \ProfileRight();
+        $profileright->update([
+            'profiles_id' => $new_tech,
+            'name'        => 'task',
+            'rights'      => 5137
+        ]);
```

Comment:
You should create the profile directly without cloning, with only the rights you need.